lemmy.yml

```diff
@@ -98,6 +98,15 @@
           - "virtualenv"
           - "python3-setuptools"
 
+    - name: Allow HTTP and HTTPS in ufw
+      when: (ansible_distribution == 'Debian') or (ansible_distribution == 'Ubuntu')
```

Comment:
As noted above, this isn't necessary as it's covered earlier in the playbook: https://github.com/LemmyNet/lemmy-ansible/blob/b6cb0ca9bb912723db9893408bd91299eeb29792/lemmy.yml#L72-L76C34